lib/modules/platform/local/index.ts

```diff
@@ -0,0 +1,123 @@
+import type { BranchStatus } from '../../../types';
+import type {
+  Issue,
+  PlatformParams,
+  PlatformResult,
+  Pr,
+  RepoResult,
+} from '../types';
+
+export const id = 'local';
+
+export function initPlatform(params: PlatformParams): Promise<PlatformResult> {
+  return Promise.resolve({
+    dryRun: 'lookup',
+    endpoint: 'local',
+    persistRepoData: true,
+    requireConfig: 'optional',
+  });
+}
+
+export function getRepos(): Promise<string[]> {
+  return Promise.resolve([]);
+}
+
+export function initRepo(): Promise<RepoResult> {
+  return Promise.resolve({
+    defaultBranch: '',
+    isFork: false,
+    repoFingerprint: '',
```

Comment:
Essentially these are all dummy values

Reply:
why not fetching the head branch of local repo?

Reply:
I want this to work in non git situations too